### PR TITLE
New features

### PR DESCRIPTION
**Pull request by DRD98**
Added new features to the code.

### CODE REVIEW

File: Fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def fibonacci(a):
```
Comment (albin-joseph) [inline Fibonacci.py:1]: @DRD98 Please change the method name to `PrintNFibanacci`

**Review by albin-joseph — CHANGES_REQUESTED** [review]
@DRD98 Please check the review comments

File: Fibonacci.py (hunk 1 above)
Comment (DRD98) [reply Fibonacci.py:1]: This name is alright.

**Review by albin-joseph — APPROVED** [review]
This branch is buildable

**Pull request merged**